velox/exec/fuzzer/ReferenceQueryRunner.h

```diff
@@ -66,6 +66,13 @@ class ReferenceQueryRunner {
     return true;
   }
 
+  /// Executes SQL query returned by the 'toSql' method based on the plan.
+  virtual std::multiset<std::vector<velox::variant>> execute(
```

Comment:
why not making this pure virtual? (` = 0;`)

Reply:
SparkQueryRunner does not currently need an implementation of this method.